### ISSUE TITLE
Add CSS grid to history section on objkt page

### ISSUE THREAD

**Issue by chrislarsc** [hicetnunc2000/hicetnunc#1326]
As a user
When I am reviewing all the transactions from the history of a given objkt
I want to read the history in the easiest way possible

It would be great if you could improve the layout of this section of this page to make it more readable.

I have included a short video with an example of what this could look like.

https://user-images.githubusercontent.com/7341675/140173000-23bf3938-26c5-4c5c-9e8b-105e2963efc2.mov

**Comment by crzypatchwork** [comment]
agreed we were intending to implement a table. did you code this? a commit in this sense is welcome also possibly with a more readable date format.

**Comment by chrislarsc** [comment]
No, I didn't code this. This was just designed. Hopefully someone else is willing to write the code. Just dropping "+00:00" from the time format would be better.
